content/pricing/index.liquid

```diff
@@ -34,80 +34,103 @@ redirect_from:
           <td class="font-mono-md text-tabular text-right">$90,000</td>
         </tr>
         <tr>
           <th scope="row">Micro</th>
+          <td class="font-mono-md text-tabular text-right">3000</td>
           <td class="font-mono-md text-tabular text-right">250</td>
           <td class="font-mono-md text-tabular text-right">$150,000</td>
         </tr>
         <tr>
           <th scope="row">Milli</th>
+          <td class="font-mono-md text-tabular text-right">4200</td>
           <td class="font-mono-md text-tabular text-right">350</td>
           <td class="font-mono-md text-tabular text-right">$210,000</td>
         </tr>
         <tr>
           <th scope="row">Centi</th>
+          <td class="font-mono-md text-tabular text-right">6000</td>
           <td class="font-mono-md text-tabular text-right">500</td>
           <td class="font-mono-md text-tabular text-right">$300,000</td>
         </tr>
         <tr>
           <th scope="row">Deci</th>
+          <td class="font-mono-md text-tabular text-right">7800</td>
           <td class="font-mono-md text-tabular text-right">650</td>
           <td class="font-mono-md text-tabular text-right">$390,000</td>
         </tr>
         <tr>
           <th scope="row">Deka</th>
+          <td class="font-mono-md text-tabular text-right">9600</td>
           <td class="font-mono-md text-tabular text-right">800</td>
           <td class="font-mono-md text-tabular text-right">$456,000</td>
         </tr>
         <tr>
           <th scope="row">Hecto</th>
+          <td class="font-mono-md text-tabular text-right">12000</td>
           <td class="font-mono-md text-tabular text-right">1000</td>
           <td class="font-mono-md text-tabular text-right">$540,000</td>
         </tr>
         <tr>
           <th scope="row">Kilo</th>
+          <td class="font-mono-md text-tabular text-right">15000</td>
           <td class="font-mono-md text-tabular text-right">1250</td>
           <td class="font-mono-md text-tabular text-right">$637,500</td>
         </tr>
         <tr>
           <th scope="row">Mega</th>
+          <td class="font-mono-md text-tabular text-right">18000</td>
           <td class="font-mono-md text-tabular text-right">1500</td>
           <td class="font-mono-md text-tabular text-right">$720,000</td>
         </tr>
         <tr>
           <th scope="row">Giga</th>
+          <td class="font-mono-md text-tabular text-right">24000</td>
           <td class="font-mono-md text-tabular text-right">2000</td>
           <td class="font-mono-md text-tabular text-right">$936,000</td>
         </tr>
       </tbody>
     </table>  
   </div>
 </section>
+<section class="usa-section">
+  <div class="grid-row grid-gap">
+    <div class="tablet:grid-col-7 usa-prose">
+      <p>
+        * All credits are available up-front when your IAA is executed. You can spend credits flexibly throughout the year if your workload is seasonal. Monthly credit amounts are provided for reference.
```

Comment:
```suggestion
        * All credits are available up-front when your IAA is executed. You can spend credits flexibly throughout the year (e.g, if your workload is seasonal, or your resource needs change during the year). Monthly credit amounts are provided for reference.
```

Reply:
How about,

> You can spend credits flexibly throughout the year if your compute needs change over time.

A little shorter, no "e.g.", combines the two examples (seasonality and changing requirements)?